### ISSUE TITLE
Lots of Twitch API Stuff missing

### ISSUE THREAD

**Issue by boostchicken** [KittehOrg/KittehIRCClientLib#278]
**What is the feature or improvement?**
Twitch IRC Api has changed a lot, I sent a PR for some stuff I need now, but there is a lot of work left to be done here.

**Comment by boostchicken** [comment]
#277

**Comment by boostchicken** [comment]
I'll keep adding stuff as I need it FYI.

**Comment by Zarthus** [comment]
is there a list of notable changes somewhere?

**Comment by boostchicken** [comment]
Nothing in a big document but if you go look at the IRC documentation you can see a lot of tags are missing and are c onditional now with raids, gifted subs, etc.  All of that is missing.

**Comment by Zarthus** [comment]
I guess if they don't have a changelog things'll lag behind, especially since it's already relatively niche

**Comment by boostchicken** [comment]
Yeah,  Twitch is notoriously bad about managing their APIs  so its a big guessing game, half the time their docs aren't even right.

**Comment by mbax** [comment]
This specific issue of Twitch stuff was resolved in 2019 and forgot to close when commits were brought in
